XbSymbolDatabase.c

```diff
@@ -494,12 +495,16 @@ void XbSymbolScanOOVPA(OOVPATable *OovpaTable,
 
         if (pFunc == pLastKnownFunc && pLastKnownSymbol == pLoop - 1) {
             //if (g_SymbolAddresses[pLastKnownSymbol->szFuncName] == 0) {
-                char output[2048];
                 sprintf(output, "Duplicate OOVPA signature found for %s, %hd vs %hd!", pLastKnownSymbol->szFuncName, pLastKnownSymbol->Version, pLoop->Version);
                 XbSymbolOutputMessage(XB_OUTPUT_MESSAGE_WARN, output);
             //}
         }
 
+        if (buildVersion < pLoop->Version) {
+            sprintf(output, "OOVPA signature is too high for [%hd] %s!", pLoop->Version, pLoop->szFuncName);
```

Comment:
nit: If we are using stdint.h (I'm not sure if we do), we should also use inttypes.h: `PRId16` although this should probably be `PRIu16` or `%hu`

Reply:
Hmm, should had been `%hu` since version is `unsigned short`.